src/intl/Core__Intl__DateTimeFormat.res

```diff
@@ -1,18 +1,120 @@
 type t
 
+type dateStyle = [#full | #long | #medium | #short]
+type timeStyle = [#full | #long | #medium | #short]
+type dayPeriod = [#narrow | #short | #long]
+type weekday = [#narrow | #short | #long]
+type era = [#narrow | #short | #long]
+type year = [#numeric | #"2-digit"]
+type month = [#numeric | #"2-digit" | #narrow | #short | #long]
+type day = [#numeric | #"2-digit"]
+type hour = [#numeric | #"2-digit"]
+type minute = [#numeric | #"2-digit"]
+type second = [#numeric | #"2-digit"]
+// Firefox also supports IANA time zone names here
+type timeZoneName = [
+  | #short
+  | #long
+]
+// TODO unsupported in node?
+// | #shortOffset
+// | #shortGeneric
+// | #longOffset
+// | #longGeneric
```

Comment:
needs decision on to include or not

Reply:
IMHO we should include it, even if not all JS runtimes support it (yet).

Reply:
@illusionalsagacity Just noticed that this is also still open. Let's include it. Maybe add a doc comment stating that it is not supported in all runtimes yet?